docker/Dockerfile

```diff
@@ -19,47 +19,58 @@
 # For those usages not covered by the GNU Affero General Public License
 # please contact with: sc_support at telefonica dot com
 
-ARG NODE_VERSION=10.19.0-slim
-FROM node:${NODE_VERSION}
+
+
+ARG NODE_VERSION=12
```

Comment:
The change from Node v10 to v12 in the Dockerfile should be noted in CHANGES_NEXT_RELEASE. For instance (inspired in https://github.com/telefonicaid/perseo-fe/releases/tag/1.13.0):

```
- Upgrade NodeJS version from 10.19.0 to 12 in Dockerfile
```

Reply:
Fixed f1dbb24